Ginger/GingerCore/ALM/RQM/ExportToRQM.cs

```diff
@@ -450,7 +450,7 @@ private ExecutionResult GetExeResultforActivityGroup(BusinessFlow businessFlow,
 
                     }
                 }
-                else if (string.IsNullOrEmpty(exeRecordId) || exeRecordId.Equals("0"))
```

Comment:
Please verify this changes if there is no TC linked in Test plan but Execution record is present